reference/test/solver/batch_bicgstab_kernels.cpp

```diff
@@ -210,13 +210,13 @@ TYPED_TEST(BatchBicgstab, ApplyLogsResAndIters)
 
     auto iter_counts = logger->get_num_iterations();
     auto res_norm = logger->get_residual_norm();
-    GKO_ASSERT_BATCH_MTX_NEAR(res.x, linear_system.exact_sol, tol * 50);
+    GKO_ASSERT_BATCH_MTX_NEAR(res.x, linear_system.exact_sol, tol * 100);
```

Comment:
This seems to get again into the range where the tolerance becomes meaningless. For bfloat16 this leads to a relative tolerance of ~0.7. But I'm also not sure how to resolve this.

Reply:
Can you maybe skip this test for bfloat16? It doesn't give us anything useful in that case.